[2025-05-05T21:53:13.893+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-05T21:53:13.928+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: google_maps_reviews.analyze_sentiment scheduled__2025-05-04T00:00:00+00:00 [queued]>
[2025-05-05T21:53:13.946+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: google_maps_reviews.analyze_sentiment scheduled__2025-05-04T00:00:00+00:00 [queued]>
[2025-05-05T21:53:13.947+0000] {taskinstance.py:2867} INFO - Starting attempt 4 of 5
[2025-05-05T21:53:13.971+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): analyze_sentiment> on 2025-05-04 00:00:00+00:00
[2025-05-05T21:53:13.993+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1718) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-05T21:53:13.997+0000] {standard_task_runner.py:72} INFO - Started process 1733 to run task
[2025-05-05T21:53:13.997+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'google_maps_reviews', 'analyze_sentiment', 'scheduled__2025-05-04T00:00:00+00:00', '--job-id', '525', '--raw', '--subdir', 'DAGS_FOLDER/***dag.py', '--cfg-path', '/tmp/tmp2p6aos3s']
[2025-05-05T21:53:14.000+0000] {standard_task_runner.py:105} INFO - Job 525: Subtask analyze_sentiment
[2025-05-05T21:53:14.079+0000] {task_command.py:467} INFO - Running <TaskInstance: google_maps_reviews.analyze_sentiment scheduled__2025-05-04T00:00:00+00:00 [running]> on host 8280c28b7f0f
[2025-05-05T21:53:14.240+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='google_maps_reviews' AIRFLOW_CTX_TASK_ID='analyze_sentiment' AIRFLOW_CTX_EXECUTION_DATE='2025-05-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-04T00:00:00+00:00'
[2025-05-05T21:53:14.243+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-05-05T21:53:14.244+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-05-05T21:53:14.245+0000] {logging_mixin.py:190} INFO - Current task name:analyze_sentiment state:running start_date:2025-05-05 21:53:13.930125+00:00
[2025-05-05T21:53:14.245+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews and current dag run status:running
[2025-05-05T21:53:14.246+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-05T21:53:14.261+0000] {base.py:84} INFO - Retrieving connection 'postgres_defaut'
[2025-05-05T21:53:14.472+0000] {logging_mixin.py:190} INFO - Sentiment analysis completed.
[2025-05-05T21:53:14.474+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-05T21:53:14.493+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-05T21:53:14.494+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=google_maps_reviews, task_id=analyze_sentiment, run_id=scheduled__2025-05-04T00:00:00+00:00, execution_date=20250504T000000, start_date=20250505T215313, end_date=20250505T215314
[2025-05-05T21:53:14.542+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-05-05T21:53:14.545+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-05-05T21:53:14.546+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews queued_at:2025-05-05 21:52:23.310129+00:00
[2025-05-05T21:53:14.548+0000] {logging_mixin.py:190} INFO - Task hostname:8280c28b7f0f operator:PythonOperator
[2025-05-05T21:53:14.580+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-05T21:53:14.638+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-05T21:53:14.642+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
